rs/src/management/mod.rs

```diff
@@ -3,9 +3,11 @@ mod errors;
 mod http_client;
 mod tunnel_locator;
 mod tunnel_request_options;
+mod policy_provider;
 
 pub use authorization::*;
 pub use errors::*;
 pub use http_client::*;
 pub use tunnel_locator::*;
 pub use tunnel_request_options::*;
+pub use policy_provider::*;
```

Comment:
`pub use` exposes the function to consumers of the crate. I think this is an implementation detail that doesn't need to be public